test/gutenberg-test-themes/block-bindings/style.css

```diff
@@ -0,0 +1,15 @@
+/*
+Theme Name: Block Bindings
+Theme URI: https://github.com/wordpress/gutenberg/
+Author: the WordPress team
+Description: Block bindings test theme.
+Requires at least: 5.3
+Tested up to: 6.7
+Requires PHP: 5.6
```

Comment:
Since WP 6.6. We only support PHP 7.2. We don't have PHP on the theme, but maybe is good to have it updated.

Reply:
Addressed in this commit: https://github.com/WordPress/gutenberg/commit/c7457d0f29f0759993ad85794721ab90896ac351